src/Subscriptions/Storage/RedisStorageManager.php

```diff
@@ -74,9 +74,11 @@ public function subscribersByTopic(string $topic): Collection
 
         return (new Collection($subscribers))
             ->map(function (?string $subscriber): ?Subscriber {
-                return is_string($subscriber)
-                    ? unserialize($subscriber)
-                    : null;
+                return
+                    is_string($subscriber)
+                    && @unserialize($subscriber) !== false
```

Comment:
We do not use error suppression through `@`. Also, calling `unserialize` twice is wildly inefficient.